src/appleseed/renderer/kernel/shading/shadingpoint.h

```diff
@@ -158,8 +158,8 @@ class ShadingPoint
     // Return the intersection point in world space, with per-object-instance ray bias applied.
     foundation::Vector3d get_biased_point(const foundation::Vector3d& direction) const;
```

Comment:
What about `direction` here, in what space is it?

Reply:
It's in world space

Reply:
![image](https://user-images.githubusercontent.com/4656466/65622840-cef21100-dfc6-11e9-8201-f4fc1cedbbf0.png)
![image](https://user-images.githubusercontent.com/4656466/65622866-db766980-dfc6-11e9-8376-73ce7585d3a6.png)
![image](https://user-images.githubusercontent.com/4656466/65622875-e3cea480-dfc6-11e9-8a96-90a78726efd4.png)

Reply:
Got it, thanks!